cas-server-core-services/src/main/java/org/jasig/cas/services/DefaultServicesManagerImpl.java

```diff
@@ -140,7 +141,7 @@ public RegisteredService findServiceBy(final long id) {
      *
      * @return the tree set
      */
-    public TreeSet<RegisteredService> convertToTreeSet() {
+    public Collection<RegisteredService> convertToTreeSet() {
```

Comment:
This is strange. The method name suggests that we get back a TreeSet, not a collection. Either the return type must be changed, or the method name.